tests/ui/type-alias-impl-trait/multi-error.stderr

```diff
@@ -10,6 +10,15 @@ note: for this opaque type
 LL |     type Bar<T> = impl Sized;
    |                   ^^^^^^^^^^
 
-error: aborting due to 1 previous error
+error[E0792]: expected generic type parameter, found `u32`
```

Comment:
unrelated: this error message does not mention "opaque types" at all. It is confusing why that restriction exists and we should include something like "when defining this opaque type, all its generic parameters must be instantiated with a generic parameter"